src/xo-app/backup/new/index.js

```diff
@@ -367,10 +372,42 @@ export default class New extends Component {
       }))
     }
 
-    // Create backup schedule.
-    return createJob(job).then(jobId => {
+    const create = () => this.setState({ redirect: true }, () => createJob(job).then(jobId => {
       createSchedule(jobId, { cron: this.state.cronPattern, enabled, timezone })
-    })
+    }))
+
+    // Create backup schedule.
+    let remoteId
+    if (job.type === 'call') {
+      const { paramsVector } = job
+      if (paramsVector.type === 'crossProduct') {
+        const { items } = paramsVector
+        forEach(items, item => {
+          if (item.type !== 'set') {
+            return false
```

Comment:
This is incorrect: you do not want to interrupt the loop if the current item does not match.